site/docs/master/contributions/minio.md

```diff
@@ -249,6 +249,20 @@ kubectl edit backupstoragelocation default -n velero
 
 Add `publicUrl: http://localhost:9000` under the `spec.config` section.
 
+## Accessing logs with an HTTPS endpoint
+
+If you're using Minio with HTTPS, you may see unintelligible text in the output of `velero describe`, or `velero logs` commands.
+
+In order to fix this, you can add a public, non-HTTPS URL to the `BackupStorageLocation`.
```

Comment:
Is it true that this URL can't be `https`? Why is that?

Reply:
The errors have usually been because the client is accessing an HTTPS endpoint with a custom certificate, so either it has to be HTTP, or the certificate has to be trusted on the client side.

I should probably clarify this wording.

Reply:
Gotcha, that makes sense.